README.md

```diff
@@ -22,6 +22,7 @@ writing your Revit Addins - use this in combination with the [RevitLookup](https
 
 ## Installation
 
+- [Installer for Autodesk Revit 2020]((https://github.com/architecture-building-systems/revitpythonshell/releases/download/2020.01.19/2020.01.19_Setup_RevitPythonShell_2020.exe)
```

Comment:
brilliant! The release is not there yet - still has to be built, but I guess I should do that soon!

Reply:
there should be an exe installer on the `Output` folder, I guess you could use that one :)